src/DataStructures/Tensor/Expressions/Product.hpp

```diff
@@ -233,3 +234,41 @@ SPECTRE_ALWAYS_INLINE auto operator*(
   return TensorExpressions::contract(
       TensorExpressions::OuterProduct<T1, T2>(~t1, ~t2));
 }
+
+/// \ingroup TensorExpressionsGroup
+/// \brief Returns the tensor expression representing the product of a tensor
+/// expression and a scalar
+///
+/// \tparam T the derived TensorExpression type of the tensor expression operand
+/// of the product
+/// \tparam ArgsList the TensorIndexs of the tensor expression operand
+/// \param t the tensor expression operand of the product
+/// \param s the scalar operand of the product
+/// \return the tensor expression representing the product of a tensor
+/// expression and a scalar
+template <typename T, typename ArgsList>
+SPECTRE_ALWAYS_INLINE auto operator*(
+    const TensorExpression<T, double, typename T::symmetry,
+                           typename T::index_list, ArgsList>& t,
+    const double s) {
+  return t * TensorExpressions::ScalarAsExpression(s);
+}
+
+/// \ingroup TensorExpressionsGroup
```

Comment:
Note that I copied and pasted the documentation from the `operator*` overload above because `\copydoc` seemed to have issues copying. My best guess for why it was not able to copy is because the template params for the `TensorExpression` operand are looked up with the `::` operator. This is what I used, which did not work:

```
/// \ingroup TensorExpressionsGroup
/// \copydoc operator*(const TensorExpression<T, double, typename
/// T::symmetry,typename T::index_list, ArgsList>&,const double)
```

I believe that if I move `T::symmetry` and `T::index_list` to default template parameters of the two new `operator*` overloads, then I think `\copydoc` would work:
```
/// documentation
/// ...
/// ...
template <typename T, typename ArgsList, typename Symmetry = T::symmetry, typename IndexList = T::index_list>
SPECTRE_ALWAYS_INLINE auto operator*(
    const TensorExpression<T, double, Symmetry, IndexList, ArgsList>& t,
    const double s) {
  return t * TensorExpressions::ScalarAsExpression(s);
}

/// \ingroup TensorExpressionsGroup
/// \copydoc operator*(const TensorExpression<T, double, Symmetry,
/// IndexList, ArgsList>&,const double)
template <typename T, typename ArgsList, typename Symmetry = T::symmetry, typename IndexList = T::index_list>
SPECTRE_ALWAYS_INLINE auto operator*(
    const double s,
   const TensorExpression<T, double, Symmetry, IndexList, ArgsList>& t) {
  return TensorExpressions::ScalarAsExpression(s) * t;
}
```

Let me know if this is preferred. The only benefit I see is not having to copy and paste the documentation. I didn't write it this way initially because I thought that a malformed call (where the symmetry of `t` is not the same as `T`) could override the default.

Reply:
Have you tried:
```cpp
// @{
/// ...
auto operator*(T,s) ...

auto operator*(s,T)....
// @}
```
This `@{`-type syntax? I always get the order of the `@` and `}` wrong so you might need to look that up